### ISSUE TITLE
[Proposal]Move docs to docs/ directory from wiki

### ISSUE THREAD

**Issue by lukasjhan** [openwallet-foundation-labs/sd-jwt-js#52]
Wiki can't manage docs version by 1.x or 2.x. So I suggest we move the documentation to docs/.

**Comment by lukasjhan** [comment]
We can use js.org like this https://github.com/lukasjhan/sdjwt

https://github.com/js-org/js.org/pull/8828

**Comment by lukasjhan** [comment]
I re-request the approval.

**Comment by lukasjhan** [comment]
It's approved https://sdjwt.js.org

**Comment by cre8** [comment]
This issue can be closed since it's merged, can't it?

**Comment by lukasjhan** [comment]
@cre8 I just make a landing page. We need to move our docs from wiki page, after we finished splitting task :)

**Comment by cre8** [comment]
I prefer the minimal approach like Markdown files in a folder on github :D Otherwise we are adding to much around the actual work

**Comment by lukasjhan** [comment]
Good points. And It can be easily migrated to another method later :)

**Comment by berendsliedrecht** [comment]
Yeah I agree with @cre8. Docusaurus has some for some cool features, but I think for a smaller, more pure, library, markdown with code snippets will suffice. Version management can likely done by something like `docs/v1/usage.md` and `https://sdjwt.js.org/v1/usage`.

**Comment by lukasjhan** [comment]
FYI: The first version of markdown docs was done in #73.
